zulia-ui-rest/src/main/java/io/zulia/ui/rest/controllers/HomeController.java

```diff
@@ -0,0 +1,46 @@
+package io.zulia.ui.rest.controllers;
+
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+
+import java.security.Principal;
+
+@Secured(SecurityRule.IS_AUTHENTICATED)
+@Controller
+public class HomeController {
+
+	@Produces(MediaType.TEXT_PLAIN)
+	@Get
+	public String home(Principal principal) {
+		System.out.println("was able to hit secured method with any role: " + principal.getName());
```

Comment:
Wouldnt be a complete PR without a system.out removal request

Reply:
lol nah, kept those on purpose until I actually implement the `HomeController` for real.